cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -113,13 +113,39 @@ pub type SignedExtra = (
 pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 
-/// Migrations to apply on runtime upgrade.
+/// All migrations that will run on the next runtime upgrade.
+///
+/// This contains the combined migrations of the last 10 releases. It allows to skip runtime
+/// upgrades in case governance decides to do so. THE ORDER IS IMPORTANT.
+#[rustfmt::skip]
+
 pub type Migrations = (
-	pallet_collator_selection::migration::v1::MigrateToV1<Runtime>,
-	pallet_multisig::migrations::v1::MigrateToV1<Runtime>,
-	InitStorageVersions,
+	migrations::V1_05_00,
+	migrations::Unreleased
 );
 
+/// The runtime migrations per release.
+#[allow(deprecated, missing_docs)]
+pub mod migrations {
+	use super::*;
+
+	pub type V1_05_00 = ();
+
+	/// New migrations should be added here in the `Unreleased` tuple.
+	/// During the release process for release `x.y.z``, the `Unreleased` migrations
+	/// will be moved to a new tuple named `Vx_y_z` and added to the migration
+	/// to execute. The `Unreleased` tuple will then be cleared.
+	/// This allows keeping track of the migrations that have been applied to the
+	/// runtime *per version*.
+	// Do not remove `#[rustfmt::skip]` or the empty tuple, even if it appears
+	// to be *temporarily* not needed. Keep a single migration per line.
+	pub type Unreleased = (
+		pallet_collator_selection::migration::v1::MigrateToV1<Runtime>,
+		pallet_multisig::migrations::v1::MigrateToV1<Runtime>,
+		InitStorageVersions,
```

Comment:
are you sure these are all unreleased and were not included in the last runtime upgrade? 

same question for the other files

Reply:
Yes, same comment than above. Without the versioning this PR brings, we cannot easily know.